app/src/main/res/values/styles.xml

```diff
@@ -13,4 +13,14 @@
         <item name="android:textSize">@dimen/text_size</item>
     </style>
 
+    <style name="NoActionBar.DarkStatusBar" parent="Theme.AppCompat.Light.NoActionBar">
```

Comment:
We can inherit from the app theme intead and just disable the action bar with:
<item name="windowActionBar">false</item>
<item name="windowNoTitle">true</item>